src/main/java/Duke.java

```diff
@@ -6,5 +8,129 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+
+        Scanner scanner = new Scanner(System.in);
+        Task[] tasks = new Task[100];
```

Comment:
You can consider the case where there are more than 100 tasks. ArrayList may be more applicable

Reply:
Moreover, 100 here is a magic number which should be avoided.